Cargo.toml

```diff
@@ -13,6 +13,7 @@ build = "build.rs"
 atty = "0.2.11"
 binary-install = "0.0.3-alpha"
 clap = "2.32.0"
+cloudflare = { git = "https://github.com/cloudflare/cloudflare-rs.git" }
```

Comment:
is there a reason for this? this will block publishing of `wrangler` to crates.io. https://crates.io/crates/cloudflare does appear to exist.

Reply:
it exists https://crates.io/crates/cloudflare

and as called out in the PR body this is blocked on a PR to that ;)

Reply:
Ah, I read it as "depends on the library" but not "depends on the specific release of the library :)"

Reply:
good catch tho, i should make an issue to, by the end of development on this feature, point at crates.io and not the git repo.

Reply:
file issue: #401